packages/macchanger.rb

```diff
@@ -0,0 +1,22 @@
+require 'package'
+
+class Macchanger < Package
+  description 'GNU MAC Changer is an utility that makes the maniputation of MAC addresses of network interfaces easier.'
+  homepage 'http://www.gnu.org/software/macchanger/'
+  version '1.7.0'
+  compatibility 'all'
+  source_url 'https://ftp.gnu.org/gnu/macchanger/macchanger-1.6.0.tar.gz'
```

Comment:
Better source? https://ftpmirror.gnu.org/macchanger/macchanger-1.6.0.tar.gz  This will localize the download to the nearest mirror.